homeassistant/components/media_player/intent.py

```diff
@@ -22,6 +25,14 @@
 DATA_LAST_PAUSED = f"{DOMAIN}.last_paused"
```

Comment:
Instead of using `hass.data`, can you make a dataclass instance, that holds this data, that is passed to the intent handlers that need this

Reply:
Done :+1: